src/FuseGPUThreadLoops.cpp

```diff
@@ -685,7 +685,7 @@ class ZeroGPULoopMins : public IRMutator {
           (op->device_api == DeviceAPI::Metal);
```

Comment:
The test above seems fragile and will likely break as soon as we add Vulkan or DirectX. Shouldn't it be `... == DeviceAPI::None) || (op->device_api != DeviceAPI::GLSL)`?